### PR TITLE
mockup-minimo

### PR DESCRIPTION
**Pull request by maycatsby**
link https://maycatsby.github.io/ui-internship-lwo2/HTML/mockup-minimo/
This website isn' fully responsive for small devices

### CODE REVIEW

File: HTML/mockup-minimo/index.html [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    <title>Mockup 3</title>
+</head>
+
+<body>
+    <header>
+        <nav class="nav">
+            <div class="logo">minim&Oslash;</div>
+            <ul class="menu">
+                <li class="menu__item">
+                    lifestyle
```
Comment (tkachukivan) [inline HTML/mockup-minimo/index.html]: probably it is link, at least add some `cursor: pointer`

File: HTML/mockup-minimo/index.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+                    photodiary
+                </li>
+                <li class="menu__item">
+                    music
+                </li>
+                <li class="menu__item">
+                    travel
+                </li>
+            </ul>
+            <span class="header__menu-icon">&#9776;</span>
```
Comment (tkachukivan) [inline HTML/mockup-minimo/index.html:32]: by bem you can't have element without block outside of block

File: HTML/mockup-minimo/index.html [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Inconsolata|Playfair+Display|Ubuntu:300,400&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+    <title>Mockup 3</title>
+</head>
+
+<body>
+    <header>
+        <nav class="nav">
```
Comment (tkachukivan) [inline HTML/mockup-minimo/index.html:16]: only menu should be in `nav` tag as logo generally is not a navigation, you can use for header class header, after have elements like header__logo/header__menu-icon, your menu would be part of nav

File: HTML/mockup-minimo/index.html [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+                <li class="menu__item">
+                    music
+                </li>
+                <li class="menu__item">
+                    travel
+                </li>
+            </ul>
+            <span class="header__menu-icon">&#9776;</span>
+        </nav>
+        <picture class="header__image">
```
Comment (tkachukivan) [inline HTML/mockup-minimo/index.html:34]: this image probably not part of header but part of first/main blog post

File: HTML/mockup-minimo/index.html [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+            <source media="(min-width: 1200px)" srcset="img/image_0@3x-min.png">
+            <source media="(min-width: 500px)" srcset="img/image_0@2x-min.png">
+            <img src="img/image_0-min.png" class="header__image" alt="article photo">
+        </picture>
+    </header>
+    <main class="container-articles">
+        <article class="article-main">
```
Comment (tkachukivan) [inline HTML/mockup-minimo/index.html:41]: if take a look at next class names you should have here `article` and modifier `article--main`

File: HTML/mockup-minimo/index.html [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+                ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
+                fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia
+                deserunt mollit anim id est laborum. Sed ut perspiciatis unde omnis iste natus error sit voluptatem
+                accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et
+                quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit
+                aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi
+                nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit,
+                sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem.
+            </p>
+            <h3 class="article__tag">leave a comment</h3>
```
Comment (tkachukivan) [inline HTML/mockup-minimo/index.html]: it is not `h3` it is probably button/link to leave a coment

File: HTML/mockup-minimo/index.html [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+        </article>
+        <section class="section-posts section-posts--first">
+            <article>
+                <picture>
+                    <source media="(min-width: 1200px)" srcset="img/image_1@3x-min.png">
+                    <source media="(min-width: 500px)" srcset="img/image_1@2x-min.png">
+                    <img src="img/image_1-min.png" class="article__image" alt="article photo">
+                </picture>
+                <h3 class="article__tag">lifestyle</h3>
```
Comment (tkachukivan) [inline HTML/mockup-minimo/index.html]: you probably need have block element tag above

File: HTML/mockup-minimo/index.html [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+        </section>
+        <section class="section-sign-in">
+            <h2 class="section-sign-in__header article__header">Sign up for our newsletter!</h2>
+            <form class="section-sign-in__form">
+                <input type="email" class="section-sign-in__input" placeholder="Enter a valid email address">
+                <input type="submit" class="section-sign-in__button" value=&rarr;>
+            </form>
+        </section>
+        <section class="section-posts">
+            <div class="section-posts--second">
```
Comment (tkachukivan) [inline HTML/mockup-minimo/index.html:130]: you can't use modifier without base class which behaviour you modify, maybe you can remove here this `div` not sure about it

File: HTML/mockup-minimo/index.html [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+                    </h2>
+                    <p class="article__paragraph">
+                        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut
+                        labore
+                        et
+                        dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+                    </p>
+                </article>
+            </div>
+            <div class="section-posts__button">
```
Comment (tkachukivan) [inline HTML/mockup-minimo/index.html:164]: I wouldn't use button inside of posts as generally it is button to load new posts but not part of them

File: HTML/mockup-minimo/index.html [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+                <button type="button" class="button">Load more</button>
+            </div>
+        </section>
+    </main>
+    <footer class="footer">
+        <div class="footer__part-left">
+            <span class="footer__span">Terms and conditions</span>
+            <span class="footer__span">Privacy</span>
```
Comment (tkachukivan) [inline HTML/mockup-minimo/index.html]: it should be links, bad names for classes, try to use something more meaningful

File: HTML/mockup-minimo/index.html [hunk 11]
```diff
@@ -0,0 +1,185 @@
...
+                    </p>
+                </article>
+            </div>
+            <div class="section-posts__button">
+                <button type="button" class="button">Load more</button>
+            </div>
+        </section>
+    </main>
+    <footer class="footer">
+        <div class="footer__part-left">
```
Comment (tkachukivan) [inline HTML/mockup-minimo/index.html]: try to name by content(what is common in every of part) for example `footer__terms` and `footer__follow`

File: HTML/mockup-minimo/style.css [hunk 12]
```diff
@@ -0,0 +1,360 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html {
+    /* 1rem = 10px  10px/16px = 0.625 = 62.5% */
```
Comment (tkachukivan) [inline HTML/mockup-minimo/style.css:8]: hm, interesting :)

File: HTML/mockup-minimo/style.css [hunk 13]
```diff
@@ -0,0 +1,360 @@
...
+}
+
+@media only screen and (max-width: 500px) {
+    html {
+        font-size: 60%;
+    }
+}
+
+body {
+    margin: 0 auto 0 auto;
```
Comment (tkachukivan) [inline HTML/mockup-minimo/style.css]: can be simplified to `0 auto`

File: HTML/mockup-minimo/style.css [hunk 14]
```diff
@@ -0,0 +1,360 @@
...
+    width: 120rem;
```
Comment (tkachukivan) [inline HTML/mockup-minimo/style.css]: it is possible to remove together with media query that page looks better and sections as footer and sign in was wide page

File: HTML/mockup-minimo/style.css [hunk 15]
```diff
@@ -0,0 +1,360 @@
...
+
+@media only screen and (max-width: 800px) {
+    .menu {
+        display: none;
+    }
+}
+
+.menu__item {
+    padding: 0;
+    align-items: center;
```
Comment (tkachukivan) [inline HTML/mockup-minimo/style.css]: probably possible to delete this, it is property for flex container [flex properties](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)

File: HTML/mockup-minimo/style.css [hunk 16]
```diff
@@ -0,0 +1,360 @@
...
+    }
+}
+
+/****** FOOTER ******/
+
+.footer {
+    margin-top: 10.3rem;
+    padding: 4rem 7.1rem 0 15rem;
+    width: 100%;
+    height: 9.7rem;
```
Comment (tkachukivan) [inline HTML/mockup-minimo/style.css]: maybe it is possible without height property

File: HTML/mockup-minimo/style.css [hunk 17]
```diff
@@ -0,0 +1,360 @@
...
+    }
+}
+
+.article__tag:last-child {
+    margin-top: 5.2rem;
+    margin-bottom: 0;
+}
+
+.article__paragraph {
+    padding-right: 0.6rem;
```
Comment (tkachukivan) [inline HTML/mockup-minimo/style.css]: unnecessary padding

**Review by tkachukivan — COMMENTED** [review]
please take a loot at comments what can be done better or where was some mistakes, keep practicing bem and it will become better, it would be great if you change page sizes for body, rest of comment is up to you

File: HTML/mockup-minimo/style.css [hunk 18]
```diff
@@ -0,0 +1,360 @@
...
+
+@media only screen and (max-width: 800px) {
+    .section-posts__button {
+        margin-top: 10rem;
+    }
+}
+
+.button {
+    padding: 2.1rem 0;
+    width: 18.2%;
```
Comment (tsvetkovv) [inline HTML/mockup-minimo/style.css:280]: let button be content-sizable
fixed width:
image
content sizeable: 
image

**Review by tsvetkovv — APPROVED** [review]

**Pull request merged**